resp/responses/configurations/strongmotion.yaml

```diff
@@ -316,6 +316,27 @@ response:
       - CUSP-200
       reversed: false
 
+  Nanometrics Dataloggers Connected to Strong Motion Sensors:
+    sensors:
+    - sensors:
+      - FBA-ES-T
+      - FBA-ES-T-ISO
+      filters:
+      - FBA-ES-T
+      channels: ZNE
+      reversed: false
+    - dataloggers:
```

Comment:
`dataloggers:` is missing

Reply:
ah yes good spot, I've added it now @ozym